ibm_catalog.json

```diff
@@ -0,0 +1,187 @@
+                  "type": "image/svg+xml"
+                },
+                "description": "This architecture supports creating and configuring an instance of watsonx Assistant."
+              }
+            ]
+          },
+          "configuration": [
+              {
+                "key": "ibmcloud_api_key"
+              },
+              {
+                "key": "use_existing_resource_group"
+              },
+              {
+                "key": "resource_group_name"
+              },
+              {
+                "key": "prefix"
+              },
+              {
+                "key": "name"
+              },
+              {
+                "key": "region",
+                "required": true,
+                "default_value": "us-south",
+                "options": [
+                  {
+                    "displayname": "Dallas (us-south)",
+                    "value": "us-south"
+                  },
+                  {
+                    "displayname": "Frankfurt (eu-de)",
+                    "value": "eu-de"
+                  },
+                  {
+                    "displayname": "London (eu-gb)",
+                    "value": "eu-gb"
+                  },
+                  {
+                    "displayname": "Sydney (au-syd)",
+                    "value": "au-syd"
+                  },
+                  {
+                    "displayname": "Tokyo (jp-tok)",
+                    "value": "jp-tok"
+                  },
+                  {
+                    "displayname": "Washington DC (us-east)",
+                    "value": "us-east"
+                  }
+                ]
+              },
+              {
+                "key": "plan",
+                "required": true,
+                "default_value": "plus-trial",
+                "options": [
+                  {
+                    "displayname": "Trial",
+                    "value": "plus-trial"
+                  },
+                  {
+                    "displayname": "Lite",
+                    "value": "free"
+                  },
+                  {
+                    "displayname": "Plus",
+                    "value": "plus"
+                  },
+                  {
+                    "displayname": "Enterprise",
+                    "value": "enterprise"
+                  },
+                  {
+                    "displayname": "Enterprise with data isolation",
+                    "value": "enterprisedataisolation"
+                  }
+                ]
+              },
+              {
+                "key": "resource_tags"
+              },
+              {
+                "key": "access_tags"
+              },
+              {
+                "key": "service_endpoints",
+                "required": true,
+                "default_value": "public",
```

Comment:
we have a default value, so may not be a required variable.
Also, did we test anytime with private endpoints?